src/makeCard.js

```diff
@@ -0,0 +1,48 @@
+export default makeCard;
+
+function makeCard(episode) {
+
+    const card = document.createElement('div')
+    card.className = "card";
+  
+    const nameDateDiv = document.createElement('div');
+    nameDateDiv.className = "name-date";
+  
+    const cardLine = document.createElement('hr');
+    cardLine.className = "line-card";
+
+    const charactersTitle = document.createElement('h4');
+    charactersTitle.innerHTML = "Personagens: ";
+  
+    const charactersDiv = document.createElement('div');
+    charactersDiv.className = "characters";
+  
+    // inserting the episode's enumeration, title and air date
+    nameDateDiv.innerHTML = `<h2>${episode.enumeration} - ${episode.title}</h2>
+                             <p>${episode.airDate}</p>`;
+    
+    // inserting characters in the list
+    let listContent = "";
+    episode.characters.forEach( (character, index, array) => {
+
+        listContent += `<li>${character}</li>`;
+        
+        // separates the characters in sets of 5
+        if ((index != 0 && (index+1)%5 == 0) || index == array.length-1) {
+            const characterList = document.createElement('ul');
+            characterList.innerHTML = listContent;
+            charactersDiv.appendChild(characterList);
+            listContent = "";
+        }
+
+    });
```

Comment:
This function will not work generally. If you have 15 characters in a episode what will happen? With 15 elements you want to have two or three lists? I guess only two, because three will break the card layout.This code just works because you have exactly 10 characters.

If you have two separate lists you want to split this list of characters in two different arrays. One with the even indexes and other with the odd ones and iterate over each this arrays creating separate lists.